media/src/App.svelte

```diff
@@ -450,6 +462,10 @@
                                     on:click={() => (currentChapter = "Linear")}>
                                     Linearization
                                 </DropdownItem>
+                                <DropdownItem
+                                    on:click={() => (currentChapter = "Hotkeys")}>
+                                    Hotkeys
```

Comment:
Can we call this "Keyboard Controls" instead of "Hotkeys"? I don't know why, Hotkeys just makes it sound supplemental, while the intention here is to have the entire application able to be controlled via keyboard as well as mouse.